astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,424 @@
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+### Expose FusionAuth Instance
+
+To make sure your local FusionAuth instance is accessible to your Laravel app, you need to [expose it to the Internet](/docs/v1/tech/developer-guide/exposing-instance). Write down the URL ngrok gave you as you'll need it soon.
+
+## Create your Laravel Application
+
+Now you are going to create a Laravel API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `app.at` cookie in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+  If you simply want to run the application, there is a completed version in the `complete-application` directory. You can install [PHP](https://www.php.net) and [Composer](https://getcomposer.org/) and use the following commands to get it up and running if you do not want to create your own.
+
+  ```shell
+  cd complete-application
+  composer install
+  ./vendor/bin/sail up -d
+  ```
+
+  You can then follow the instructions in the "Run the API" section.
+</Aside>
+
+To create the application from scratch, you'll use [Laravel Sail](https://laravel.com/docs/10.x/installation#laravel-and-docker) to set up and run your application using Docker.
+
+```shell
+curl "https://laravel.build/fusionauth-quickstart-php-laravel-api?with=mariadb" | bash
+cd fusionauth-quickstart-php-laravel-api
+```
+
+### Initialize the Application
+
+Initialize the Laravel application using the following:
+
+```shell
+./vendor/bin/sail up
+```
+
+### Update Routes
+
+Change `routes/api.php` to add both `/make-change` and `/panic` endpoints for your ChangeBank application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/routes/api.php"} lang="php"/>
+
+### Add Security
+
+Include FusionAuth's library to add JWT support to the authentication system.
+
+```shell
+./vendor/bin/sail composer require fusionauth/jwt-auth-webtoken-provider web-token/jwt-signature-algorithm-rsa
```

Comment:
I get an error when I try to run this.

`> @php artisan package:discover --ansi

   UnexpectedValueException 

  Invalid route action: [App\Http\Controllers\ChangeBank\PanicController].

  at vendor/laravel/framework/src/Illuminate/Routing/RouteAction.php:92
     88▕      */
     89▕     protected static function makeInvokable($action)
     90▕     {
     91▕         if (! method_exists($action, '__invoke')) {
  ➜  92▕             throw new UnexpectedValueException("Invalid route action: [{$action}].");
     93▕         }
     94▕ 
     95▕         return $action.'@__invoke';
     96▕     }

  i   `App\Http\Controllers\ChangeBank\PanicController` was not found: Controller class `App\Http\Controllers\ChangeBank\PanicController` for one of your routes was not found. Are you sure this controller exists and is imported correctly? 

      +9 vendor frames 

  10  routes/api.php:21
      Illuminate\Routing\RouteRegistrar::__call()
      +4 vendor frames 

  15  app/Providers/RouteServiceProvider.php:34
      Illuminate\Routing\RouteRegistrar::group()

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
fusionauth-quickstart-php-laravel-api (main): 
`
![Screenshot 2023-10-24 at 4 09 34 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/9ad12880-680e-4a70-9995-8ea1766ab51b)

Reply:
Thanks, I've changed the order to put this before creating routes.